### ISSUE TITLE
Various tables overflow their section container

### ISSUE THREAD

**Issue by fedde-s** [opentargets/platform#946]
Since release 20.02, the contents of many of the table-containing sections on the profile pages will display outside of their sections when viewed in a narrow window. Examples include the **RNA Expression** section on the association profile page (Expression Atlas), the **Protein information** section on the target profile page (Uniprot), and the **Drugs** section on all profile pages. I believe the issue to have been introduced by commit https://github.com/opentargets/webapp/commit/72f735a5c0815e5861cc5708381654a589203a55.

![Screenshot of the Drugs section overflowing its container](https://user-images.githubusercontent.com/4929431/77766927-9ba94100-7040-11ea-8cfe-da06db3e5a70.png)

**Comment by fedde-s** [comment]
From the commit message of the commit I reference, I'm not sure why this behaviour was introduced. Is the reason for it still relevant, or would it make sense to revert the commit as a bugfix?

**Comment by fedde-s** [comment]
I see—doing that would cut of the large tooltip of the **Gene prioritisation score** column heading in the **Genetic associations** / **Common diseases** table. Might there be a way to make the overflow apply _only_ to that tooltip?

**Comment by LucaFumis** [comment]
Hi Fedde,
well spotted - you're correct in all your assumptions/findings.
The change was introduced to allow the (somewhat large) tooltip in the genetic evidence table.
I was aware at lower resolution the drugs table would present some rendering issues, but then again the summary charts would break anyway. I don't think at the time I picked up on the other tables.
It _might_ be possible to revert the change and only fix the genetics evidence table to allow the tooltip - I'd imagine I'd tried that the time, but can have another look...

**Comment by fedde-s** [comment]
[This article](https://css-tricks.com/popping-hidden-overflow/) led me to try out what would happen if the popover had no positioned ancestors within the `.scroll-table-panel .table-panel`—which worked for me, and made the popover visible even without `overflow: visible`.

…unfortunately, the closest positioned ancestor is positioned by datatables.min.css:
```
table.dataTable thead .sorting,
table.dataTable thead .sorting_asc,
table.dataTable thead .sorting_desc,
table.dataTable thead .sorting_asc_disabled,
table.dataTable thead .sorting_desc_disabled {
	cursor: pointer;
	position: relative;
}
```

**Comment by andrewhercules** [comment]
@fedde-s: we are planning for our 20.04 release, which should be available w/c Monday 27 April, and we can look at resolving this issue then. :-)

**Comment by LucaFumis** [comment]
Been playing with the css without much luck. Tentatively trying positioning of the popover below the element as an alternative approach, but haven't fully checked it doesn't brake anything else.

As a quick and safe solution (since we're rewriting the app) we could shorten the text of that specific tooltip...

**Comment by andrewhercules** [comment]
@LucaFumis, could we try shortening the tooltip to the following?

`The gene prioritisation score is the output from our Locus2Gene pipeline. For more information on the score and our pipeline, click here.`

Does that resolve the overflow issue?

**Comment by andrewhercules** [comment]
Issue closed as work completed as part of 20.04 release process
